distribution/pom.xml

```diff
@@ -27,7 +27,10 @@
 
     <name>hazelcast-distribution</name>
     <artifactId>hazelcast-distribution</artifactId>
-    <packaging>pom</packaging>
```

Comment:
Won't this cause a jar being generated?

Reply:
Good point. The jar will be generated, but we don't want to deploy it. So I'm adding a new commit with a change in `maven-deploy-plugin` configuration.